packages/dev-stack/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@omakase/dev-stack",
+  "version": "0.0.1",
+  "description": "A collection of devDependencies commonly used across Artsy projects",
+  "repository": "https://github.com/omakase-js/omakase/tree/master/packages/dev-stack",
+  "author": "Artsy Inc.",
+  "license": "MIT",
+  "dependencies": {
+    "@babel/cli": "^7.4.4",
+    "@babel/core": "^7.4.5",
+    "@babel/plugin-proposal-class-properties": "^7.4.4",
+    "@babel/plugin-proposal-decorators": "^7.4.4",
+    "@babel/plugin-proposal-json-strings": "^7.2.0",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
+    "@babel/plugin-syntax-import-meta": "^7.2.0",
+    "@babel/preset-env": "^7.4.5",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/preset-typescript": "^7.3.3",
+    "@omakase/cli": "^0.0.10",
+    "babel-jest": "^24.8.0",
+    "babel-plugin-lodash": "^3.3.4",
+    "babel-plugin-module-resolver": "^3.2.0",
+    "babel-plugin-relay": "^4.0.0",
+    "babel-plugin-styled-components": "^1.10.0",
+    "enzyme": "^3.9.0",
+    "enzyme-adapter-react-16": "^1.13.1",
+    "husky": "^2.3.0",
+    "jest": "^24.8.0",
+    "jest-junit": "^6.4.0",
+    "jest-raw-loader": "^1.0.1",
+    "lint-staged": "^8.1.7",
+    "patch-package": "^6.1.2",
+    "postinstall-prepare": "^1.0.1",
+    "prettier": "^1.17.1",
+    "relay-compiler": "^4.0.0",
+    "relay-compiler-language-typescript": "^4.3.0",
+    "tslint": "^5.16.0",
+    "tslint-config-prettier": "^1.18.0",
+    "tslint-plugin-relay": "^0.0.3",
+    "tslint-react": "^4.0.0",
+    "typescript": "^3.4.5",
+    "typescript-styled-plugin": "^0.14.0"
+  },
+  "devDependencies": {
+    "graphql": "^14.3.1",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "styled-components": "^4.3.0"
+  }
```

Comment:
Various `peerDep` needs from above `dependencies` array.

Reply:
Seems like a large overlap with the deps provided by the runtime-stack, perhaps you could depend on that here instead of repeating the list?

Reply:
Oh hmm, it looks like this still triggers the `peerDep` warnings. Thoughts?

Reply:
Ah, ok, I guess peer dep warnings are only satisfied by direct dependencies. Unfortunate, but so be it